wemake_python_styleguide/constants.py

```diff
@@ -311,15 +312,15 @@
 
 #: Approximate constants which real values should be imported from math module.
 APPROXIMATE_CONSTANTS: Final = frozenset((
-    3.14,
-    3.141,
-    3.1415,
-    6.28,
-    6.282,
-    6.283,
-    2.71,
-    2.718,
-    2.72,
+   round(pi, 2),
```

Comment:
Sorry, I was not clear enough. That's not how it should be. Let me show you the API we need:

```python
APPROXIMATE_CONSTANTS: Final = frozenset((
    math.pi,
    math.e,
    math.tau,
))
```

And it is a visitors responsibility to find that some numbers are very close to these values. Because, this solution is hard to scale. Imagine, that we can have ten's of constants.